prow/git/v2/adapter.go

```diff
@@ -44,12 +44,12 @@ type clientFactoryAdapter struct {
 
 // ClientFromDir creates a client that operates on a repo that has already
 // been cloned to the given directory.
-func (a *clientFactoryAdapter) ClientFromDir(org, repo, dir string) (RepoClient, error) {
+func (a *clientFactoryAdapter) ClientFromDir(org, repo, cloneURI, dir string) (RepoClient, error) {
 	return nil, errors.New("no ClientFromDir implementation exists in the v1 git client")
 }
 
 // Repo creates a client that operates on a new clone of the repo.
-func (a *clientFactoryAdapter) ClientFor(org, repo string) (RepoClient, error) {
+func (a *clientFactoryAdapter) ClientFor(org, repo, cloneURI string) (RepoClient, error) {
 	r, err := a.Client.Clone(org, repo)
 	return &repoClientAdapter{Repo: r}, err
 }
```

Comment:
It doesn't seem great that we are ignoring the cloneURI in this implementation. Is this old client still used? If so, do we still need the old and new git clients to have the same interface? Should we migrate remaining uses of the old client to the new client?

Reply:
I'm not sure about the usage of this old client, it feels to me another PR, I can add a TODO here

Reply:
added a TODO